src/MockClassifier.Api/Controllers/InstitutionController.cs

```diff
@@ -15,20 +28,30 @@ public class InstitutionController : ControllerBase
         [HttpPost]
         public AcceptedResult Post([FromBody] MessagesInput messages)
         {
-            var minsitries = new List<string>();
-
-            // TO DO
```

Comment:
These comments are probably redundant now I think because the code has been implemented? Same applies to the other comment inside the `foreach` loop

Reply:
Agreed. have actually removed these comments as this has been implemented